src/main/java/frc/robot/Constants.java

```diff
@@ -21,9 +22,18 @@
  */
 public final class Constants {
 
+    public static int kP;
+    public static int kI;
+    public static int kD;
+
     public static int LEDPort = 0;
     public static int LEDLength = 10;
 
     public static Port colorSensorPort = Port.kOnboard;
     public static float proximityThreshhold = 200;
+
+    public static int driveStickPort = 0;
```

Comment:
These are good for now with testing with 2019, but we will eventually need to change them to adapt to the new drivetrain with neos.